frontend/hooks/useBuildLogs.js

```diff
@@ -7,6 +7,8 @@ const initResultsState = {
   offset: 0,
   logs: [],
   state: '',
+  isLoading: true,
```

Comment:
This is good; I thought about this after our review yesterday and we should codify this pattern for data loaders